src/main/java/com/movie_collection/gui/models/MovieModel.java

```diff
@@ -1,40 +1,46 @@
 package com.movie_collection.gui.models;
 
 import com.google.inject.Inject;
-import com.movie_collection.be.Category;
 import com.movie_collection.be.Movie;
-import com.movie_collection.bll.services.interfaces.ICategoryService;
 import com.movie_collection.bll.services.interfaces.IMovieService;
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
 
 import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
 
-public class MovieModel implements IMovieModel{
+public class MovieModel implements  IMovieModel{
+
     private final IMovieService movieService;
 
-   // private ObservableList<Movie> moviesList = FXCollections.observableArrayList();
+    private final ObservableList<Movie> allMovies;
```

Comment:
do we really need to lists like that ?

Reply:
this way was faster for me when I cleaned the filter query - otherwise it has to get the movies from DB every time we clear filter which will be slower